### ISSUE TITLE
Addition of new diffusion models that are VERY ACCURATE (beats DeepAR)

### ISSUE THREAD

**Issue by cody-craig** [Nixtla/neuralforecast#496]
Timegrad is a relatively new model that uses Autoregressive Denoising Diffusion for Multivariate Probabilistic Time Series Forecasting Link to paper [here](https://arxiv.org/pdf/2101.12072.pdf) . It has shown to outperform many types of time series models over several datasets. Recently a group of Chinese researchers Were able to create a model called D3VAE that uses diffusion, denoising, and disentanglement to beat Timegrad model. Link to that paper [here](https://openreview.net/pdf?id=rG0jm74xtx)

The Problem is both of these models are not very user friendly. I have experienced many hours of pain trying to implement these models and others have had the same problems.

IF Nixtla can implement these models in an easy to use, open source way, that would be a game changer for me personally. Thanks a bunch, keep up the good work

<img width="715" alt="Screen Shot 2023-03-31 at 7 26 20 AM" src="https://user-images.githubusercontent.com/88689204/229131327-f242e25b-742d-4a6c-ac7
<img width="580" alt="Screen Shot 2023-03-31 at 7 55 08 AM" src="https://user-images.githubusercontent.com/88689204/229131345-d9580ecf-cbe7-4791-9fb2-3b3d383a94bc.png">
d-1b8f06a8e56b.png">

**Comment by kyr0** [comment]
+1

**Comment by marcopeix** [comment]
I don't think this is a relevant addition anymore. Feel free to reopen if you think otherwise, I'm assuming the other models now outperform this one.
